src/services/cssCompletion.ts

```diff
@@ -156,7 +162,8 @@ export class CSSCompletion {
 	}
 
 	private getPropertyProposals(declaration: nodes.Declaration, result: CompletionList): CompletionList {
-		let properties = languageFacts.cssDataManager.getProperties();
+		const triggerPropertyValueCompletion = this.settings && this.settings.triggerPropertyValueCompletion;
```

Comment:
@octref should I need to check if config exists (undefined) and if it’s undefined return true and if it’s exist take a value? Because right now if we do not pass settings to ‘configure’ or they are undefined, empty, it wouldn’t ‘retrigger’.

Reply:
You can provide a fallback in the configuration setting (in `css-language-features`) so it's never undefined.
But I agree, `true` should be the default if no setting is found.